installers/installer.sh

```diff
@@ -77,7 +55,16 @@ function __ubuntu_prerequisites() {
 }
 
 function __rhel_prerequisites() {
-    __centos_prerequisites "$1"
+    local sync_gateway=$1
+    yum update -q -y
+    if [[ "$OS_VERSION" == 8* ]]; then
+        yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm -q
+        yum install jq net-tools python2 wget -q -y
+        python2 -m pip -q install httplib2
+        return
+    fi
+    yum install epel-release jq net-tools python2 wget -q -y
```

Comment:
I'm not sure I understand the point of duplicating all this in the branch above for a slightly different `epel-release`.
Would it not be better to have the branch do `epel-release` then a standard install and run python2 for both?

Reply:
I was trying to keep the install into a single line (if possible) but i can refactor

Reply:
I mean you have the same line twice, just on RHEL 8 you have an extra RPM for the repo